freqtrade/exchange/deribit.py

```diff
@@ -0,0 +1,67 @@
+"""Deribit exchange subclass"""
+
+import logging
+from datetime import datetime
+from typing import Dict, List, Optional, Tuple
+
+from pandas import DataFrame
+
+from freqtrade.enums import MarginMode, TradingMode
+from freqtrade.exchange import Exchange
+
+
+logger = logging.getLogger(__name__)
+
+
+class Deribit(Exchange):
+    _params: Dict = {"trading_agreement": "agree"}
+    _ft_has: Dict = {
+        "stoploss_on_exchange": True,
+        "stop_price_param": "triggerPrice",
+        "stop_price_prop": "triggerPrice",
+        "stoploss_order_types": {"limit": "limit", "market": "market"},
+        "order_time_in_force": ["GTC", "IOC", "FOK", "PO"],
+        "ohlcv_candle_limit": 720,  # TODO
+        "ohlcv_has_history": False,  # TODO
```

Comment:
If this is true (doesn't provide ohlcv history) - then i don't think it'll make sense for freqtrade to support it.

From experience i can tell - exchanges without history are not being used frequently - as it doesn't allow users to backtest these exchanges.
As such, it becomes time wasted for no good reason as barely anyone will use it.

Reply:
this is probably not the correct value, I've done very little for this exchange so far, but there's someone else who might help contribute with me so I created a draft

Reply:
so essentially, this is an empty, mostly wrong shell?

Reply:
correct